app/src/renderer/components/common/ShortAddress.vue

```diff
@@ -0,0 +1,79 @@
+<template lang="pug">
+.short-address
+  .address(@click.prevent.stop="copy" v-tooltip.top="address")#address {{ shortAddress }}
+  .success(:class="{active:showSuccess}")
+    i.material-icons check
+    span Copied
+</template>
+
+<script>
+import { clipboard } from "electron"
+export default {
+  name: `short-address`,
```

Comment:
I'd rename the component to `short-bech32` since we are using it for addresses and pubkeys

Reply:
i think this is a good suggestion!

Reply:
we can change the name later tho, nbd

Reply:
Good suggestion.

Reply:
Yes, I agree.

Reply:
agree that i should rename it or agree that it can wait until later? (or both)

Reply:
I think they were talking about renaming the component

Reply:
I think we should rename it. We can so later.